### PR TITLE
Support multibranch jobs after Jenkins 2.266

### PR DESCRIPTION
**Pull request by eoc-ross**
Fixing broken compatiblity with Jenkins v2.268 with an ugly hack. Related jenkins issue https://issues.jenkins.io/browse/JENKINS-65608.

### CODE REVIEW

File: src/main/java/org/jvnet/hudson/plugins/nextbuildnumber/NextBuildNumberAction.java [hunk 1]
```diff
@@ -78,8 +78,8 @@ private void checkPermission(Job job) {
             MultiBranchProject parentProject = (MultiBranchProject) job.getParent();
             try {
                 parentProject.getACL().checkPermission(getPermission());
-            } catch (org.acegisecurity.AccessDeniedException e) {
-                if (parentProject.getParent() instanceof ComputedFolder) {
+            } catch (Exception e) {
```
Comment (akomakom) [inline src/main/java/org/jvnet/hudson/plugins/nextbuildnumber/NextBuildNumberAction.java:81]: Perhaps we could just switch to [hasPermission](https://javadoc.jenkins-ci.org/hudson/security/ACL.html#hasPermission-hudson.security.Permission-)?

Comment (eoc-ross) [reply src/main/java/org/jvnet/hudson/plugins/nextbuildnumber/NextBuildNumberAction.java:81]: I like the multicatch way but I'm having troubles with spring security dependency.

```
[ERROR] updateShouldFailIfJobDoesNotExist(org.jvnet.hudson.plugins.nextbuildnumber.CommandTest)  Time elapsed: 0.28 s  <<< ERROR!
java.lang.NoSuchMethodError: org.springframework.beans.factory.support.RootBeanDefinition.<init>(Ljava/lang/Class;Z)V
	at hudson.util.spring.DefaultBeanConfiguration.createBeanDefinition(DefaultBeanConfiguration.java:201)
	at hudson.util.spring.DefaultBeanConfiguration.getBeanDefinition(DefaultBeanConfiguration.java:180)
	at hudson.util.spring.DefaultBeanConfiguration.addProperty(DefaultBeanConfiguration.java:217)
	at hudson.util.spring.BeanBuilder.setProperty(BeanBuilder.java:551)
```

I'll look into it more tomorrow, for now I'm off.

**Comment by akomakom** [comment]
I think it's safe to remove, according to the docs.  I'd just test it locally for whatever multibranch types you have installed.

**Comment by eoc-ross** [comment]
It works fine with https://plugins.jenkins.io/cloudbees-bitbucket-branch-source/

**Comment by akomakom** [comment]
@eoc-ross confirmed that it also works with GitHub Branch Source

**Review by akomakom — APPROVED** [review]

**Comment by akomakom** [comment]
@eoc-ross did you intend to close?

**Comment by akomakom** [comment]
No worries

**Comment by akomakom** [comment]
Thanks @eoc-ross ! 
I'll do some cleanup and make 1.7

**Comment by akomakom** [comment]
Version 1.7 is now available and seems to work.

**Pull request merged**